src/microservices/users/tests/UnitTests/Api/Tests/Handlers/UserHandlerV1Tests.cs

```diff
@@ -2,17 +2,66 @@
 {
     using System.Threading.Tasks;
     using Xunit;
-    using Google.Protobuf.WellKnownTypes;
     using FluentAssertions;
-    using Grpc.Core;
     using global::Api.Protos;
     using global::Api;
     using System;
     using UnitTests.Api.Helpers;
     using Microsoft.Extensions.Logging;
-    using Moq;
+    using FakeItEasy;
+    using UnitTests.Common;
+    using global::Application.UseCases.Interfaces;
 
     public class UserHandlerV1Tests
     {
+        private readonly ILogger<UserHandlerV1> stubbedLogger;
+        private readonly IGetUserUseCase stubbedGetUserUseCase;
+
+        public UserHandlerV1Tests()
+        {
+            this.stubbedLogger = A.Fake<ILogger<UserHandlerV1>>();
```

Comment:
I think we should avoid initializing fake objects in constructor or setup/tear down method. It makes the test less readable.

Reply:
Also we must agree on convention, how do we name the objects. `stubbedLogger` or `loggerStub`, that will reflect to other variables as well, for instance `getUserIdRequestStub` and `stubbedGetUserIdRequest`

Reply:
Yeah, I'll add all the points which we want to discuss in the document. I will remove the constructor and initialize the fake objects in each unit test.